tests/Jellyfin.Common.Tests/Json/JsonOmdbConverterTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+using MediaBrowser.Common.Json;
+using MediaBrowser.Common.Json.Converters;
+using MediaBrowser.Providers.Plugins.Omdb;
+using Xunit;
+
+namespace Jellyfin.Common.Tests.Json
+{
+    public static class JsonOmdbConverterTests
```

Comment:
I would add at least 1 test where the string isn't `N/A` (and prob also one where the string is `null`)